functions/common/debezium-autoconfigure/README.adoc

```diff
@@ -0,0 +1,305 @@
+# Debezium Auto-Configuration
```

Comment:
This might cause the italic problem.
This is not MarkDown, but ASCIIDoc, so probably `=` should be used for head lines.
Although it renders in the view OK 🤷